kitsune/notifications/tests/test_api.py

```diff
@@ -80,7 +81,7 @@ def test_correct_fields(self):
         eq_(serializer.data['action_object']['type'], 'question')
         eq_(serializer.data['action_object']['id'], q.id)
         eq_(serializer.data['target'], None)
-        eq_(type(serializer.data['timestamp']), datetime)
+        datetime.strptime(serializer.data['timestamp'], '%Y-%m-%dT%H:%M:%SZ')
```

Comment:
I'm not sure what this line is doing. Is it verifying that the data can be parsed with that template? If so, maybe add a comment like:

```
# If this data isn't in the right form, then this will throw an exception.
```